client/tests/integration/smartcontracts/executor_with_admin/src/lib.rs

```diff
@@ -22,7 +22,8 @@ struct Executor {
 }
 
 fn visit_instruction(executor: &mut Executor, authority: &AccountId, isi: &InstructionBox) {
-    if parse!("admin@admin" as AccountId) == *authority {
+    // crate::integration::upgrade::ADMIN_ID
+    if *authority == parse!(AccountId, "ed012076E5CA9698296AF9BE2CA45F525CB3BCFDEB7EE068BA56F973E9DD90564EF4FC@admin") {
```

Comment:
In attempt to share multihash between the test and the executor module, `parse!` macro was modified to accept `expr`, but no use cases atm